### ISSUE TITLE
Training yolov2 with backend mobilenet on WIDER FACE dataset. val_mAP is very small 0.008

### ISSUE THREAD

**Issue by jps892** [experiencor/keras-yolo2#440]
I am training yolov2 with backend mobilenet on WIDER FACE dataset. val_mAP is very small 0.008 and val_loss is also not decreasing.

If object is very small, will changing grid size help? and can we change grid size for yolov2 with backend mobilenet?

Is there any pre-processing annotated dataset (resizing to 224x244 of image and anchor) required before training?

If you have trained yolov2 with mobilenet backend with any standard dataset please post the result.

here is loss, val_loss and val_mAP pattern: 

![Screenshot from 2019-11-29 13-34-10](https://user-images.githubusercontent.com/29859992/69857649-61a87800-12b6-11ea-9958-0d7cb0d90b4a.png)
![Screenshot from 2019-11-29 13-34-39](https://user-images.githubusercontent.com/29859992/69857685-708f2a80-12b6-11ea-85f2-d62d49c87328.png)

**Comment by badstones** [comment]
I carry out mobilenet+yolov2 on VOC2007, the mAP is about 30% which is much lower than mobilenet2+yolov3, I don't know why
